src/components/HostStatus/getHostStatus.js

```diff
@@ -0,0 +1,3 @@
+import * as _ from 'lodash';
+
+export const getHostStatus = host => _.get(host, ['metadata', 'labels', 'metalkube.org/operational-status']);
```

Comment:
this should probably be in `utils/selectors` as it is not a component. Or make a file  `utils/hostSelectors`,  if you think there will be more of them

Reply:
I have taken an inspiration for this in VmStatus/getVmStatus.js selector which is colocated to the component. I expect this selector to grow in a similar manner and a statu component to be added. But yeah, I am happy to move it under utils as you're saying.

Reply:
`getVmStatus.js` does a bit more than just data selection and it belongs to `VmStatus` component. Nevertheless some of the functionality from `getVmStatus.js` could be refactored to some selectors file.

Reply:
Done